.github/workflows/sonar-cloud-static-analysis.yml

```diff
@@ -27,15 +27,6 @@ jobs:
         with:
           java-version: 11
 
-        # The sonar cache appears to be about ~450 MB
-      - name: Cache SonarCloud packages and analysis
-        uses: actions/cache@v2
-        id: sonarcloud-cache
-        with:
-          path: ~/.sonar
-          key: ${{ runner.os }}-sonar-${{ env.SONAR_SCANNER_VERSION }}
-          restore-keys: ${{ runner.os }}-sonar-${{ env.SONAR_SCANNER_VERSION }}
-
       - name: Download and set up sonar-scanner
```

Comment:
Typo in the second commit message's body: "agian" => "again" :)

Reply:
fixed, thanks!